clang-tools-extra/test/clang-doc/mustache-index.cpp

```diff
@@ -0,0 +1,74 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: clang-doc --format=mustache --output=%t --executor=standalone %s 
+// RUN: FileCheck %s < %t/GlobalNamespace/index.html
+
+enum Color {
+  RED,
+  BLUE,
+  GREEN
+};
+
+class Foo;
+
+// CHECK:       <li class="sidebar-section">
+// CHECK-NEXT:      <a class="sidebar-item" href="#Enums">Enums</a>
+// CHECK-NEXT:  </li>
+// CHECK-NEXT:  <ul>
+// CHECK-NEXT:      <li class="sidebar-item-container">
+// CHECK-NEXT:          <a class="sidebar-item" href="#{{[0-9A-F]*}}">enum Color</a>
+// CHECK-NEXT:      </li>
+// CHECK-NEXT:  </ul>
+// CHECK:       <li class="sidebar-section">
+// CHECK-NEXT:      <a class="sidebar-item" href="#Classes">Inner Classes</a>
+// CHECK-NEXT:  </li>
+// CHECK-NEXT:  <ul>
+// CHECK-NEXT:      <li class="sidebar-item-container">
+// CHECK-NEXT:          <a class="sidebar-item" href="#{{[0-9A-F]*}}">Foo</a>
+// CHECK-NEXT:      </li>
+// CHECK-NEXT:  </ul>
+
+// CHECK:       <section id="Enums" class="section-container">
+// CHECK-NEXT:      <h2>Enumerations</h2>
+// CHECK-NEXT:      <div>
+// CHECK-NEXT:          <div id="{{[0-9A-F]*}}" class="delimiter-container">
+// CHECK-NEXT:              <div>
+// CHECK-NEXT:                  <pre>
+// CHECK-NEXT:                      <code class="language-cpp code-clang-doc">
+// CHECK-NEXT:                          enum Color
+// CHECK-NEXT:                      </code>
+// CHECK-NEXT:                  </pre>
+// CHECK-NEXT:              </div>
+// CHECK-NEXT:              <table class="table-wrapper">
+// CHECK-NEXT:                  <tbody>
+// CHECK-NEXT:                  <tr>
+// CHECK-NEXT:                      <th>Name</th>
+// CHECK-NEXT:                      <th>Value</th>
+// CHECK:                  </tr>
```

Comment:
Does the tag not line up becacuse you deleted `-NEXT` or do we emit it with really strange indentation?